packages/jest-runtime/src/index.ts

```diff
@@ -1823,6 +1821,11 @@ export default class Runtime {
       throw moduleNotFoundError;
     }
 
+    e.message = `Jest: Got error evaluating ${path.relative(
```

Comment:
It was impossible to see where errors originated without this

Before: ![image](https://user-images.githubusercontent.com/1404810/111864816-7903ec80-8963-11eb-8656-43801ecc742a.png)

After: ![image](https://user-images.githubusercontent.com/1404810/111864839-9b960580-8963-11eb-844f-4530a261c8ce.png)

Reply:
let's not have that now though to reduce the surface of this PR

Reply:
Oh wow I ran into that reference error yesterday for some reasons and it was gone out of the blue